chartsLibrary/SolidStepChart.tsx

```diff
@@ -0,0 +1,159 @@
+import { onMount } from "solid-js";
+import { LineChart } from "@carbon/charts";
+import { ChartConfig, LineChartOptions } from "@carbon/charts/interfaces";
+
+const data = [
```

Comment:
No data should be in library components

Reply:
That was just a mistake, dad. Just a mistake.